### DISCUSSION TITLE
Is there a quick way to reload an SFZ file?

### DISCUSSION THREAD

**Discussion by skybrian** [falkTX/Carla#1675] [Q&A] [answered]
I edited an SFZ file using an external editor to fix a problem with a sample's loop parameters. The playback doesn't change in Carla unless I remove the SFZ component and add it back again. Is there a better way to make it reload the file?

(Canvas->Refresh didn't do it.)

**Comment by falkTX** [comment]
the backend supports it, but there is nothing on the GUI to click to trigger it.

for sfz handling I recommend to use https://github.com/sfztools/sfizz/ anyway, which supports this and many other things.

**Reply by skybrian** [reply]
I tried sfizz but couldn't get it to work. The VST3 plugin doesn't appear in Carla's list. The LV2 plugin shows up in Carla, but segfaulted.

**Comment by skybrian** [comment, marked as answer]
I tried the AU for the sforzando plugin and that worked for me. It has a "refresh" menu item to reload the SFZ file.
